test/pytest/test_utils.py

```diff
@@ -58,12 +58,15 @@ def torchserve_cleanup():
     delete_all_snapshots()
 
 
-def register_model(model_name, url):
+def register_model(model_name, url, batch_size=1, initial_workers=1):
     params = (
         ('model_name', model_name),
         ('url', url),
-        ('initial_workers', '1'),
+        ('initial_workers', str(initial_workers)),
         ('synchronous', 'true'),
+        ('batch_size', str(batch_size)),
+        ('batch_delay', '10000')
+        
```

Comment:
This part should not be changed if the old registerModel  (see following) is kept. 
public static void registerModel(
            Channel channel,
            String url,
            String modelName,
            boolean withInitialWorkers,
            boolean syncChannel)

Reply:
my bad - was under the impression that you can add arguments to the end of a function with optional values to make sure no backwards compatbility is broken